sycl/doc/GetStartedGuide.md

```diff
@@ -463,7 +463,7 @@ command:
 
 ### Obtain prerequisites for ahead of time (AOT) compilation
 
-[Ahead of time compilation](design/CompilerAndRuntimeDesign.md#ahead-of-time-aot-compilation)
+[Ahead of time compilation](design/CompilerAndRuntimeDesign.html#ahead-of-time-aot-compilation)
```

Comment:
The original link is correct. Doxygen is expected to convert relative links to Markdown documents to HTML versions. We need to figure out why the conversion doesn't work.
This change breaks the link in Markdown version of the document: https://github.com/intel/llvm/blob/sycl/sycl/doc/GetStartedGuide.md.